src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/model/OffenderBooking.java

```diff
@@ -177,6 +193,167 @@
     @Column(name = "ADMISSION_REASON")
     private String admissionReason;
 
+    @OneToOne(mappedBy = "booking", cascade = CascadeType.ALL)
+    @PrimaryKeyJoinColumn
+    private ReleaseDetail releaseDetail;
```

Comment:
interesting this is a OneToOne, so for a booking with mutliple releases and recalls this just has the latest details ?

Reply:
Yes correct!